### ISSUE TITLE
Division Issue int typecast

### ISSUE THREAD

**Issue by atharvap** [EmilienDupont/wgan-gp#1]
The self.img_size[0]/16 needs to be typecasted to int:

`class Generator(nn.Module):
    def __init__(self, img_size, latent_dim, dim):
        super(Generator, self).__init__()

        self.dim = dim
        self.latent_dim = latent_dim
        self.img_size = img_size
        self.feature_sizes = (int(self.img_size[0] / 16), int(self.img_size[1] / 16))`

**Comment by SummerColdWind** [comment]
My pytorch version: `2.8.0+cu128`
`model.py` Line 14: `self.feature_sizes = (int(self.img_size[0] / 16), int(self.img_size[1] / 16))`
`model.py` Line 72: `output_size = 8 * dim * (img_size[0] // 16) * (img_size[1] // 16)`
`training.py` Line 98: `self.losses['gradient_norm'].append(gradients.norm(2, dim=1).mean().item())`
`training.py` Line 44: `self.losses['GP'].append(gradient_penalty.item())`
`training.py` Line 54: `self.losses['D'].append(d_loss.item())`
`training.py` Line 71: `self.losses['G'].append(g_loss.item())`
